.changeset/ninety-books-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Closes [#1505](https://github.com/cloudflare/wrangler2/issues/1505) by extending `wrangler tail` to allow for passing worker routes as well as worker script names.
```

Comment:
Maybe an example? Something like "`wrangler tail example.com/*` will work if you have a worker assigned to the route example.com/*`